### DISCUSSION TITLE
How could we stop to export a part of metrics to collector?

### DISCUSSION THREAD

**Discussion by aaron-ai** [open-telemetry/opentelemetry-java#4702] [Q&A] [answered]
There are two kinds of value in the label0 of my counter.

data0: {topic: "foo"}, value: 3
data1: {topic: "bar"}, value: 5

Now the "bar" is deprecated, how could we stop to export the data1 to collector?

**Comment by jkwatson** [comment, marked as answer]
I believe you can accomplish this by configuring a View into your SDK that removes the attribute. See here: https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/ViewBuilder.java#L72
